docs/docs/pods.md

```diff
@@ -12,16 +12,15 @@ Currently, Marathon pods can only be created and administered via the `/v2/pods/
 
 - Co-located containers.
 - Pod-level resource isolation.
-- Sandbox and ephemeral volumes that are shared among pod containers.
-- Configurable sandbox mount point for each container.
+- Pod-level sandbox and ephemeral volumes.
 - Pod-level health checks.
 
 # Quick Start
 
 1. Run the following REST call, substituting your IP and port for `<ip>` and `<port>`:
 
     ```bash
-    $ http POST <ip>:<port>/v2/pods <<EOF
+    $ curl POST <ip>:<port>/v2/pods <<EOF
```

Comment:
Thanks for noticing that. I've changed the arguments, but still need to test them all